pulp_rpm/app/serializers.py

```diff
@@ -306,16 +306,25 @@ class RpmRepositorySerializer(RepositorySerializer):
     """
 
     metadata_signing_service = serializers.HyperlinkedRelatedField(
-        help_text="A reference to an associated signing service.",
+        help_text=_("A reference to an associated signing service."),
         view_name='signing-services-detail',
         queryset=AsciiArmoredDetachedSigningService.objects.all(),
         many=False,
         required=False,
         allow_null=True
     )
+    package_retention_policy = serializers.IntegerField(
```

Comment:
Completely open to renaming this

Reply:
Yeah, I'm +1 to rename but I don't have good suggestions :D

For the sake of brainstorming ideas:

latest_package_count
package_recency_count
recent_package_count
keep_recent_packages
retain_package_count

Reply:
+1 to ```keep_recent_packages``` or (Ina's suggestion) ```retain_package_versions```

Reply:
@ipanova @ggainey @dkliban @goosemania @daviddavis  Any strong preferences between those two (or a different one)?

Reply:
How about `retained_package_versions`? keep_ and retain_ are verbs, but what you're really specifying is a noun, "how many package-versions should I keep in this repo". Does that make sense?

Reply:
I think it could be considered either a noun or a verb from the user perspective.  It's a fine suggestion, though.  I don't really want to bike shed this forever, maybe we should just set up a poll and whichever one wins is the one we go with.

Reply:
Good idea - maybe a quick 24-hr poll, with my, ipanova's, and your suggestions? Note that I am not strongly tied to my suggestion - any of those three would be fine I think, if you picked one and just went with it I'd be fine.